src/SymfonyCorp/Connect/OAuthConsumer.php

```diff
@@ -149,9 +155,9 @@ public function getEndpoint()
         return $this->endpoint;
     }
 
-    public function getBrowser()
+    public function getHttpClient()
```

Comment:
if unused, should be removed (and if used, let's double check it's legit and remove if not)